modules/installation-guide/partials/proc_installing-che-on-minikube-using-chectl.adoc

```diff
@@ -14,11 +14,22 @@ This section describes how to install {prod-short} on Minikube using {prod-cli}.
 
 .Procedure
 
-* Run the following command:
+* To install {prod-short} in multi-user mode (default and recommended), run the following command:
```

Comment:
This is a getting started, we should keep things simple. Since it's the default and recommended we should not go through much of the details. I would not change this line.

Reply:
ok, I'll revert this line